frontend/components/category-headings.spec.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import { shallow } from 'enzyme'
 import { expect } from 'chai'
-import { CategoryHeading, CategoryHeadings } from './category-headings'
 import { H3 } from './common'
+import { CategoryHeading, CategoryHeadings } from './category-headings'
```

Comment:
I would've expected the previous order (`./cat..` before `./com...`) but guess I should re-read the rule (I assume eslint made all these changes)

Reply:
I think this rule doesn't alphabetize, unfortunately.

Reply:
ahhh that makes so much more sense. i was seriously questioning my mental sorting abilities